### PR TITLE
Player movement rendering

### PR DESCRIPTION
**Pull request by NickHackman**
## Features
- Fixed issues with platform specific paths
- Made paths absolute
- Added Player class
  - Player can move right
  - Player can move up "jump"
- Added Autopep8 for formatting used by editors

## TODO
- Screen updating causes buggy flickering **epilepsy warning**
- Player jump physics
- Discuss on how we want to manage movement
  - whether the player should move or everything else should just move towards the player
- Image path and asset for player

### CODE REVIEW

File: src/game/engine/game.py [hunk 1]
```diff
@@ -3,34 +3,45 @@
...
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 pygame.quit()
                 sys.exit()
             if event.type == pygame.KEYDOWN:
                 if event.key == pygame.K_ESCAPE:
                     running = False
-        
+                if event.key == pygame.K_RIGHT:
+                    player.right()
```
Comment (jklypchak13) [inline src/game/engine/game.py:36]: If my understanding of the game is right, we'll never really need this as a core mechanic, right? If this just for debugging?

File: src/game/engine/game.py [hunk 2]
```diff
@@ -3,34 +3,45 @@
...
+                if event.key == pygame.K_SPACE:
+                    player.jump()
```
Comment (jklypchak13) [inline src/game/engine/game.py:38]: Should we only do spacebar, or should we allow for arrow/WASD controls?

File: src/game/engine/player.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    position: Tuple[int, int, int, int]
+         starting (x, y, width, height) of player
+
+    image_path: str
+         path to image to use as player image
+    """
+
+    # position: x, y, width, height
+    position: Tuple[int, int, int, int]
```
Comment (jklypchak13) [inline src/game/engine/player.py:21]: Does it makes sense to store width/height with position?

In the end we probably want the following vectors/tuples
- position: (X,Y)
- velocity: (xVel,yVel) (For jump moving on the screen/jump physics)
- acceleration: (xAccel, yAccel) (small xAccel for scaling difficulty, yAccel for jump physics)
- dimensions: (Width,Height)

**Review by jklypchak13 — APPROVED** [review]
Seems good. Just some small notes for the future.

File: src/game/engine/game.py (hunk 1 above)
Comment (NickHackman) [reply src/game/engine/game.py:36]: cool, yeah just proof of concept

File: src/game/engine/player.py (hunk 3 above)
Comment (NickHackman) [reply src/game/engine/player.py:21]: Pygames uses it, so I did too! See `player.py:draw`

**Pull request merged**